src/Materials/shadowDepthWrapper.ts

```diff
@@ -81,7 +81,7 @@ export class ShadowDepthWrapper {
      */
     constructor(baseMaterial: Material, scene: Scene, options?: IIOptionShadowDepthMaterial) {
         this._baseMaterial = baseMaterial;
-        this._scene = scene;
+        this._scene = scene ?? Engine.LastCreatedScene;

```

Comment:
Looks like this causes a Typescript error: TS2304: Cannot find name 'Engine'.

Reply:
Sorry, I missed adding the import. Fixed.